houdini/python2.7libs/hda/takes.py

```diff
@@ -21,10 +20,82 @@
 
 
 def _to_parm_name(take_name):
-    """Prefix with take_ to avoid parm name clashes."""
+    """Prefix take name with take_ to avoid parm name clashes."""
     return "take_%s" % take_name
 
 
+def _is_takable(template):
+    """Determine if template is takable.
+
+    It is takable if it has a tag `takes` set to the string
+    True, or if it is a label. We enable all labels because
+    labels don't have tags so we can't differentiate.
+
+    """
+    return (template.type().name() == "Label" or
+            template.tags().get("takes") == "True")
```

Comment:
Nit: you could use `strtobool` here: https://docs.python.org/2/distutils/apiref.html#distutils.util.strtobool

Reply:
Only problem is strtobool doesn't like 'NoneType' : (object has no attribute 'lower') - so I'd have to test that first. 
template.tags() returns a dict of str, str. It can't be numeroc or bools. 

One other option is to just test for presence of the key - but I'd still have to put something in the value.